ProteoformSuiteInternal/TopDownProteoform.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Proteomics;
+
+namespace ProteoformSuiteInternal
+{
+    public class TopDownProteoform : Proteoform
+    {
+        public string uniprot_id { get; set; }
+        public string name { get; set; }
+        public string sequence { get; set; }
+        public int start_index { get; set; }
+        public int stop_index { get; set; }
+        public List<Ptm> ptm_list { get; set; } = new List<Ptm>();
```

Comment:
Could you use a PtmSet here, and maybe make the variable name localized_ptm_set or top_down_ptmset. It should be distinct from the ptm_set that's going to be hosted in Proteoform (coming soon 😅 ).